File: src/main/java/no/nav/domain/pensjon/regler/grunnlag/Eksportrett.java
package no.nav.domain.pensjon.regler.grunnlag;

import no.nav.domain.pensjon.regler.kode.EksportlandCti;

import java.io.Serializable;

public class Eksportrett implements Serializable {



    /**
     * Angir om personen har eksportrett eller ikke.
     */
    private boolean eksportrett;

    /**
     * Angir land personen bor i.
     */
    EksportlandCti bostedsland;

}

File: src/main/java/no/nav/pensjon/regler/domain/grunnlag/Eksportrett.kt
package no.nav.pensjon.regler.domain.grunnlag

import no.nav.pensjon.regler.domain.kode.EksportlandCti
import java.io.Serializable

class Eksportrett : Serializable {
    /**
     * Angir om personen har eksportrett eller ikke.
     */
    var isEksportrett = false

    /**
     * Angir land personen bor i.
     */
    var bostedsland: EksportlandCti? = null

}